src/ui/hooks/useRoomViewModel.ts

```diff
@@ -134,13 +131,14 @@ export class TRRoomViewModel extends ViewModel implements IRoomViewModel {
       timeline,
       tileClassForEntry: this._tileClassForEntry,
     });
-    this._timelineVM = this.track(
-      new TimelineViewModel(
-        this.childOptions({
-          tileOptions: this._tileOptions,
-          timeline,
-        })
-      )
+
+    // NOTE: We are not tracking TimelineViewModel to dispose
+    // Cause when it dispose it also closes timeline of current room
```

Comment:
Is this going to cause a memory leak?

Reply:
It safe. We are just keeping timeline open for rest of our session and if a room get's removed from session `BaseRoom.js`'s dispose method also dispose its timeline.